packages/test-config/README.md

```diff
@@ -0,0 +1 @@
+# @leather.io/test-config
```

Comment:
How about just calling this `@leather.io/test` so we can use it for more things in future? 

 Maybe even shared test code itself, mocks and the extension test-app?

You should add some content to the README too